resources/views/components/tables/desktop/exchanges.blade.php

```diff
@@ -16,7 +16,7 @@ class="text-right"
                 breakpoint="md-lg"
                 responsive
             >
-                <span>({{ Settings::currency()}})</span>
+                <span>(USD)</span>
```

Comment:
Can this use the lang for now?

```suggestion
                <span>(@lang('currencies.usd.currency'))</span>
```